Finish mobile version

abel-blue opened the pull request:
## I have added
- skeleton for grids, cards and form
- add styles for the form and cards

adel-gu commented on style.css:
```diff
@@ -200,3 +200,255 @@
+  background: whitesmoke;
+}
+
+.card {
+  margin: 30px;
+  background: #ebf0ee;
+  height: 350px;
+  width: 310px;
+  border-radius: 20px;
+}
```
- [ ] All skills cards are supposed to fill the whole screen size as shown in the Figma design, Please stick to the design as much as possible, and also consider fixing the spacing between each card.

### Your Design
![image](https://user-images.githubusercontent.com/68030297/219818483-3f2c0e2d-894c-4155-bd31-89fc5ca3b7dd.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/219818655-d377d70d-8f29-42d6-bbf6-7e7b5718e5b7.png)

- To simply achieve that, I suggest removing both `margin and the width` then to fix the left and the right space for the card simply use `margin-inline` which is a shortcut for left+right margins.

- And for the vertical spacing between each card since you are using the grid system, in the parent class `abt-card` you can benefit by using the `gap` property that will set the above and under space for each card.

adel-gu reviewed (changes requested):
# Status: REQUIRED CHANGES ♻️ 

**Hi @abel-blue 👋🏼** 

Good job so far! 👍🏼 you really did well implementing the project requirements😎 however, There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- All linter checks are passing ✔️ 
- HTML/CSS best practices are followed ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

---
### You can also consider:

- [x] A good detailed and descriptive PR summary that is specifically describing the achievements and changes you implemented in this Milestone will always make The PR stand out more✨, I highly recommend working on that.

![image](https://user-images.githubusercontent.com/68030297/219819600-d19b0247-b884-4971-bd6f-6847e7e621b6.png)
---

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

abel-blue commented:
## I have added
- style on resolutions of range min 375px, max 786px instead for h1, cards, grids as depict on Figma
- fixed the spacing between each card
- good detailed and descriptive PR summary

RaoAkif reviewed (approved):
## STATUS: APPROVED 🎉🎉🎉

Hi @abel-blue, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![approve-image](https://ci5.googleusercontent.com/proxy/zq80mRZVpq-eAhdzmvNYP-LJmESV4U2mCLkUDj-J77H6FKB6oLO-eL8kGFENlnk15aQQPnJ2K7hSGZiaZBHMKFZEM6yCAT5H0leJxszpiJ_ZRxNc5L9uv6Wf0BFZUomZYiYN6ppiHgltD6qjc_Kh8RbH3b4nVLPghJlpfxM29IAi_q8djloxFmcQQ_I2iIY_rYXVUeBiSfNsVRfk0iq3wo75_nf9QHejNs5EwcDwp7n4vukAV0rFF6xBnAenIBlJ9_CuSGN37pZ0FDROc_LU_rInohwgD5sTJnITUxeXsq5juhu5btEu0AhtqpFV5MDVTQMfZw=s0-d-e1-ft#https://camo.githubusercontent.com/b4e8f035e0f2d20302034402026b8df675e3a9dd93b722d1b0a047112e68044d/68747470733a2f2f632e74656e6f722e636f6d2f4a52665739444c534f764141414141432f7365616c2d7365616c2d6f662d617070726f76616c2e676966)

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
